include/wx/gtk/webview_webkit.h

```diff
@@ -134,6 +138,18 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewWebKit : public wxWebView
     //create-web-view signal and so we need to send a new window event
     bool m_creating;
 
+    //variables used for Find()
+    int m_findFlags;
+    wxString m_findText;
+    int m_findPosition;
+    int m_findCount;
+
+#if wxUSE_WEBVIEW_WEBKIT2
+    //Used for webkit2 extension
+    GDBusServer *m_dbus_server;
+    GDBusProxy *m_extension;
+#endif
```

Comment:
That crossed my mind too, and searching for them in the cpp and h files that you modified shows `m_findCount`, `m_dbus_server` and `m_extension`, each of them in one such situation. And that IMHO they would look better as getters. And at least the other 3 could co back to being protected.